<?xml version="1.0" ?>

<robot name="table">

    <link name="table_frame">
        <visual>
            <origin xyz="0 0 0"/>
            <geometry>
                <box size="2 2 0.1"/>
            </geometry>
            <material name="steel">
                <color rgba="0.5 0 0 1"/>
            </material>
        </visual>0
        <collision>
            <origin xyz="0 0 0"/>
            <geometry>
                <box size="2 2 0.1"/>
            </geometry>
            <material name="">
                <contact name="steel"/>
            </material>
        </collision>
        <inertial>
            <mass value="10"/>
            <inertia ixx="100" ixy="0.0" ixz="0.0" iyy="100" iyz="0.0" izz="100"/>
            <origin xyz="0 0 0"/>
        </inertial>
    </link>

</robot>